use_case_guide/multi_tenant.rst

```diff
@@ -0,0 +1,517 @@
+
+Applications do not need to undergo downtime during shard rebalancing. Read requests continue seamlessly, and writes are locked only when they affect shards which are currently in flight.
+
+::
+
+  -- Spread data evenly between the nodes
+
+  SELECT rebalance_table_shards('companies');
+
+As it executes, the command outputs notices of each shard it moves:
+
+.. code-block:: text
+
+  NOTICE:  00000: Moving shard [id] from [host:port] to [host:port] ...
+
+Refreshing the Nodes tab in the Cloud Console shows that the new node now contains data! It can now help processing requests for some of the tenants.
+
+Dealing with Big Tenants
+------------------------
+
+.. note::
+
+  This section uses features available in Citus Cloud and Citus Enterprise, and will not work in the community edition.
+
+The previous section describes a general-purpose way to scale a cluster as the number of tenants increases. However, users often have two questions. The first is what will happen to their largest tenant if it grows too big. The second is what are the performance implications of hosting a large tenant together with small ones on a single worker node, and what can be done about it.
+
+Regarding the first question, investigating data from large SaaS sites reveals that as the number of tenants increases, the size of tenant data typically tends to follow a `Zipfian distribution <https://en.wikipedia.org/wiki/Zipf%27s_law>`_.
+
+.. image:: ../images/zipf.png
+
+For instance, in a database of 100 tenants, the largest is predicted to account for 23% of the data. In a more realistic example for a large SaaS company, if there are 10k tenants, the largest will account for ~2% of the data. Even at 10TB of data, the largest tenant will require 200GB, which can pretty easily fit on a single node.
+
+Another question is regarding performance when large and small tenants are on the same node. Standard shard rebalancing will improve overall performance but it may or may not improve the mixing of large and small tenants. The rebalancer simply distributes shards to equalize storage usage on nodes, without examining which tenants are allocated on each shard.
+
+To improve resource allocation and make guarantees of tenant QoS it is worthwhile to move large tenants to dedicated nodes. Citus provides the tools to do this.
+
+In our case, let's imagine that our old friend company id=5 is very large. We can isolating the data for this tenant happens in two somewhat low-level steps. We'll present the commands here, and you can consult :ref:`tenant_isolation` for the theory behind them.
+
+First consolidate the tenant's data into a bundle (called a shard) suitable to move. The CASCADE option also applies this change to the rest of our tables distributed by :code:`company_id`.
```

Comment:
We're actually not consolidating data. We're just isolating the tenants data.

So, can we say "First, separate the tenant's data into its own shard".